core/tools/dnsmasq/bootstrap.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Exit if something goes wrong
+set -e
+
+# Dnsmasq provides DHCP and DNS servers for small networks
+echo "Installing dnsmasq."
+apt install -y --no-install-recommends dnsmasq
+
+echo "Configuring dnsmasq."
+CONFIG_ORIG_PATH="/home/pi/tools/dnsmasq/dnsmasq.conf"
+CONFIG_DEST_PATH="/etc/dnsmasq.conf"
+
+# Checks if the installation process indeed created the original file,
+# preventing changes in this behavior to silent affect our application.
+[ ! -f "${CONFIG_DEST_PATH}" ] && (
```

Comment:
Since the important thing here is the binary itself, we may need just to check the binary installation or if the binary itself exists.